src/userscript.ts

```diff
@@ -10,7 +10,9 @@ const loadLSSM = () => {
 
     script.src = `${host}core.js?_=${new Date().getTime()}&uid=${
         I18n.locale
-    }-${user_id}`;
+    }-${user_id}&branch=${
+        localStorage.getItem(`${prefix}_branch`) ?? 'stable'
```

Comment:
What's the prefix by default?

Reply:
it is `lssmv4` and will be set when building the userscript.